test/onnx/test_fx_op_consistency.py

```diff
@@ -1548,6 +1538,11 @@ def skip_torchlib_forward_compatibility(
             "Reshape", "empty tensor"
         ),
     ),
+    xfail(
+        "nn.functional.instance_norm",
+        model_type=pytorch_test_common.TorchModelType.TORCH_EXPORT_EXPORTEDPROGRAM,
```

Comment:
cc @titaiwangms, similar to what @liqunfu observed the other day, I need to move this xfail downwards to here to correctly only xfail on 'TORCH_EXPORT_EXPORTEDPROGRAM'.

Reply:
Tracked by https://github.com/pytorch/pytorch/issues/120918.
I will use this to see the case.

Reply:
Revisiting on this. I think with matcher seems reasonable to put it down here though.

Reply:
I just confirmed that model_type only works with subfails.